build/agent/Dockerfile

```diff
@@ -0,0 +1,53 @@
+FROM nginx:1.22.1-alpine
+ARG AGENT_VERSION
+ARG ALPINE_VERSION
+
+WORKDIR /agent
+
+RUN apk add --no-cache libcap
+
+# For now, get the agent apk package from github release. Eventually, we will pull the pre-build package from nginx.org.
+RUN wget -nv -O agent.apk https://github.com/nginx/agent/releases/download/v$AGENT_VERSION/nginx-agent-$AGENT_VERSION-v$ALPINE_VERSION-x86_64.apk \
+    && apk add --allow-untrusted agent.apk
+
+# Copy nginx-agent config file and entrypont script.
+# We could also mount this to the Pod.
+COPY ./build/agent/nginx-agent.conf /etc/nginx-agent/nginx-agent.conf
+COPY ./build/agent/entrypoint.sh /agent/entrypoint.sh
+
+# Copy nginx config file and httpmatches njs module.
+# We could also mount this to the Pod.
+COPY ./internal/nginx/modules/src/httpmatches.js /usr/lib/nginx/modules/njs/httpmatches.js
+COPY ./build/agent/nginx.conf /etc/nginx/nginx.conf
+
+# Create nginx directories.
+RUN mkdir -p /etc/nginx/secrets /var/lib/nginx /var/log/nginx
+
+# Clear /conf.d directory.
+RUN rm -f /etc/nginx/conf.d/*
```

Comment:
Can we collapse multiple RUN instructions? This will follow best practicies -- https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#minimize-the-number-of-layers -- minimize number of layers

Reply:
Sure.